stools/clair/__init__.py

```diff
@@ -95,58 +145,78 @@ def help(retval=0):
         version()
         sys.exit(0)
 
+    if args.report is None and args.report_format is not None:
+        print("\nWarning: --report-format has no effect when --report is not passed\n")
+
     # Generate Clair controller
     clair = Clair(args.clair_host, args.clair_port)
     if not clair.ping():
         sys.exit(1)
 
     print(clair)
-    
+
     # Local Server
-    webroot = '/var/www/images'
+    webroot = "/var/www/images"
 
     # Start the server and serve static files from root
-
     if args.server is True:
-        print('\n1. Starting server...')
-        server = 'http://%s:%s/' %(args.host, args.port)
+        print("\n1. Starting server...")
+        server = "http://%s:%s/" % (args.host, args.port)
         process = Process(target=start, args=(args.port, args.host, webroot))
-        # start(port=args.port, host=args.host, static_folder=webroot)
```

Comment:
Please leave this line, it’s there to show how to debug manually from inside the container.

Reply:
Reverted :+1: